client/js/verification-portal/pregnancy-resource-center/form.js

```diff
@@ -8,6 +8,7 @@ import VerifiedCheckbox from '../components/verified-checkbox'
 import Button from '../../components/button'
 import Heading from '../../components/heading'
 import Services from './services'
+import Toggle from '../../components/toggle'
```

Comment:
This is great for testing, but let's not merge any of this. I've tested it so you can go ahead and revert this file.